app/controllers/idv/review_controller.rb

```diff
@@ -56,6 +57,7 @@ def create
         success: true,
         fraud_review_pending: idv_session.profile.fraud_review_pending?,
         fraud_rejection: idv_session.profile.fraud_rejection?,
+        gpo_verification_pending: idv_session.profile.gpo_verification_pending?,
```

Comment:
Are we checking `idv_session.profile.gpo_verification_pending?` in some places and `current_user.gpo_verification_pending_profile?` in others because the information hasn't yet been put into the session (for the latter case)?

Reply:
Here we check idv_session because the profile was just created a few lines up. In other places we check current user because they may be in a new session where the data isn't persisted.